.github/workflows/update-metadata.yml

```diff
@@ -0,0 +1,28 @@
+name: "Update metadata"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 0'
+
+permissions: { }
+jobs:
+  update-members:
+    name: "Update members.json"
+    runs-on: ubuntu-latest
+    if: github.repository == 'hibernate/.github'
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v6
+      with:
+        fetch-depth: 1
+    - name: Configure for commits
+      run: |
+        git config --global user.name 'GitHub Actions'
+        git config --global user.email 'ci@hibernate.org'
+    - run: ./scripts/update-members.sh
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN_FOR_METADATA_UPDATE }}
```

Comment:
This is an org secret I already created, with permission to look up team members.